src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/con-2061-email-templates-recipients.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN TRANSACTION
```

Comment:
This file should be named something like 'con-2061-create-email-template-recipients' and then it should clear the values in the [EmailTemplates].Recipients column when it has finished

Reply:
done